src/features/yamlEditor/components/editor.tsx

```diff
@@ -52,6 +52,7 @@ export const CodeEditor = ({ code, onChangeEditor }: ICodeEditor) => {
         theme={isGrayscaleStyleType ? undefined : greenAccentTheme}
         extensions={[StreamLanguage.define(yamlLanguage)]}
         onChange={e => convertToJSON(e)}
+        data-testid="yaml-editor"
```

Comment:
Should have done this a long time ago since it allows us to avoid the text matching (`page.get_by_text`) that you can find in the diff on the left in the Playwright test file. But I wonder if this test id data attribute should be "yaml-editor-container" or maybe "code-mirror-container" because, as you might notice in the changes to the Playwright test file in this PR, the locator for the actual textbox field looks like this:

```py
page.get_by_test_id("yaml-editor").get_by_role("textbox")
```

That's because the HTML structure of the CodeMirror React component is like this:

```html
<div data-testid="yaml-editor">
  <!-- stuff -->
  <div contenteditable>
  <!-- more stuff -->
```